action/protocol/protocol.go

```diff
@@ -49,21 +48,11 @@ type Committer interface {
 	Commit(context.Context, StateManager) error
 }
 
-// Aborter performs abort action of the protocol
-type Aborter interface {
-	Abort(context.Context, StateManager) error
-}
-
```

Comment:
this is for protocol to perform necessary actions (if there's any) in case attempted commit block failed, don't think we need to remove it